releasenotes/notes/add-examples-to-characterization-experiments-e77d4d26c6b49694.yaml

```diff
@@ -0,0 +1,8 @@
+---
+features:
+  - |
+    Added minimal working code examples to the API pages for each experiment,
+    especially in characterizaion experiments. The minimal working code example
+    is a code snippet which users can copy and paste to run the experiment.
+    Users are required to specify a backend in the code example. As a default,
+    the backend is a simulator such as GenericBackendV2(num_qubits=27).
```

Comment:
```suggestion
    Users are required to specify a backend to use the code example. By default,
    the backend used in the examples is a simulator such as `FakeManilaV2`.
```

Reply:
Yes! Thank you for correcting this point.

Reply:
@coruscating, thank you for your review.

I added the shot option, such as shots=1000, in order to save the running time of code examples on the IBM Quantum real devices. As for the data frame option, it looks beautiful. I agree to your suggestion.
I have updated the branch local/issue-#1238 following your suggestion.

For QubitSpectroscopy, exp.set_experiment_options(amp=0.005) may be an appropriate choice of the parameter.
I also find a bug in /qiskit_experiments/library/characterization/__init__.py, EFSpectroscopy is missing.

I want to push the branch to origin/issue-#1238.
By the way, my reno report number is 0.5.0-… appeared in the html file created locally.